.tokenami/tokenami.config.ts

```diff
@@ -303,11 +303,11 @@ export default createConfig({
 			/* eslint-enable perfectionist/sort-objects */
 		},
 		z: {
-			"0": 0,
-			"10": 10,
-			"100": 100,
-			"1000": 1_000,
-			"10000": 10_000,
+			"0": "0",
+			"10": "10",
+			"100": "100",
+			"1000": "1000",
+			"10000": "10000",
```

Comment:
theme values have to be strings now bcos the `css` utility checks for `number` type to wrap with `calc`. this was the simplest solution to avoid a runtime dependency on your theme config (to check if the property passed is a grid property).